src/components/InputWithLabel.js

```diff
@@ -19,9 +22,16 @@ const InputWithLabel = (props) => {
         value = {props.todoTitle} 
         onChange = {props.handleTitleChange} 
         ref = {inputRef}
+        className={styles.InputField}
     ></input>
     </>
   );
 };
 
-export default InputWithLabel;
\ No newline at end of file
+InputWithLabel.propTypes = {
```

Comment:
I think the children prop type needs to updated - I'm getting this console error when I run the app
![Screenshot 2023-04-21 at 4 39 06 PM](https://user-images.githubusercontent.com/5691923/233744689-504f6a87-bb4d-4a95-8f79-59dc774b6805.png)

Reply:
Thanks for this Mark.

I don't have any error when I ran it locally on my machine.

Just thinking what could be the cause...

Reply:
I will look into it please.

Reply:
Oh bummer! I wouldn't worry about it then, typically when I use prop types, I use the PropTypes.node for children - this accounts for basically anything that can be rendered - strings, components, numbers, etc. 

I'll approve!